tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java

```diff
@@ -56,6 +56,13 @@ public void TestLanguageNameCoverage() {
         // We add in additionsToTranslate below the set in attributeValueValidity.xml $languageExceptions
         // (both sets are included in SDI.getCLDRLanguageCodes() but we do not use that until later).
         Set<String> additionsToTranslate = ImmutableSortedSet.of("zxx", "mul",
+
+            // TODO: fix this test (or related code such as Validity.getInstance().getCodeToStatus),
+            // which fails due to 25 locales moved from seed to common
+            // "aa", "bal", "bgn", "blt", "bss", "cad", "cch", "cic", "cu", "hnj", "ken", "lij", "osa", "prg", "quc",
+            // "sdh", "sid", "sma", "smj", "ssy", "szl", "trw", "vec", "vo", "wbp",
+            // Reference: https://unicode-org.atlassian.net/browse/CLDR-6396
+
```

Comment:
better to have the test ignore non-basic locale ids

Reply:
i don't think it's sustainable to add former-seed localesto this test

Reply:
could remove all elements of temp2 that are non basic locales

Reply:
this pr fixes the other tests and leaves this test unfixed

how about if I remove this file from the pr while we're figuring out what to do about this test?

Reply:
i've just done that

Reply:
done